packages/wonder-blocks-core/components/with-ssr-placeholder.js

```diff
@@ -22,14 +22,27 @@ type Props = {|
      * the server-side renderer and the rehydration -- or it defeats
      * the purpose of using the WithSSRPlaceholder component.
      */
-    placeholder: ?(() => ?React.Node),
+    placeholder: ?() => ?React.Node,
```

Comment:
This is super sketchy, I'm surprised that prettier does this.

Reply:
Agreed. I preferred what I wrote.